src/ontology/imports/uberon_import.owl

```diff
@@ -172,8 +171,8 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/UBERON_0000122> "neural fiber bundle")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/UBERON_0000122> "UBERON:0000122")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/UBERON_0000122> "neuron projection bundle")
-SubClassOf(<http://purl.obolibrary.org/obo/UBERON_0000122> <http://purl.obolibrary.org/obo/CARO_0000000>)
```

Comment:
@ddooley did you manually change this subclass to not use CARO? Because in Uberon this's still the case. We need to change it in Uberon instead.

Reply:
Ah!  Yes I manually disconnected it, not realizing that the connection was in uberon_import.owl rather than ro-edit.owl  . I can reestablish it (redo uberon_import.owl), but that would temporarily leave http://purl.obolibrary.org/obo/CARO_0000000 dangling in RO, if that's ok with folks?

Reply:
Yes, it's not a problem. It would be good if you could make an issue in Uberon to make this change.

Reply:
Could you run again `sh run.sh make imports/uberon_import.owl` to properly update the uberon_import, please?

Reply:
I think I did yesterday eve.  It changed uberon_import.owl .  I noticed it didn't put the CARO term back in as parent but I assumed that was perhaps because the CARO term isn't imported anymore separately in caro_import.owl

Reply:
You're right.